### PR TITLE
ci: Collect tracing logs

### LINKED ISSUE #2027: Collect tracing logs
Enable to collect the tracing logs for debugging purposes.

### PR DESCRIPTION
**Pull request by GabyCT**
This will enable to collect the tracing logs for debugging purposes.

Fixes #2027

Signed-off-by: Gabriela Cervantes <gabriela.cervantes.tellez@intel.com>

### CODE REVIEW

File: .ci/teardown.sh [hunk 1]
```diff
@@ -172,6 +177,9 @@ collect_logs()
 		echo "Kernel Log:"
 		sudo journalctl --no-pager -t kernel
 
+		echo "Runtime Tracing Log:"
+		sudo journalctl --no-pager -t kata-runtime
```
Comment (chavafg) [inline .ci/teardown.sh]: here you are getting the kata-runtime log... which is already collected above.
I think you'll need to collect what is inside `/var/log/kata-tests/traces`.
See: https://github.com/kata-containers/tests/blob/master/tracing/tracing-test.sh#L20

**Comment by GabyCT** [comment]
@chavafg changes applied

File: .ci/teardown.sh [hunk 2]
```diff
@@ -92,6 +96,9 @@ collect_logs()
 
 		[ "${have_collect_script}" = "yes" ] && sudo -E PATH="$PATH" "${collect_script_path}" > "${collect_data_log_path}"
 
+		tracing_log_directory="/var/log/kata-tests/logs/traces/"
+		cat "${tracing_log_directory}/${tracing_log_filename}" > "${tracing_log_path}"
```
Comment (devimc) [inline .ci/teardown.sh]: Q, why not just `cp "${tracing_log_directory}/${tracing_log_filename}" "${tracing_log_path}"`?

**Pull request merged**